Sources/FileSystem/FileSystem.swift

```diff
@@ -78,12 +78,6 @@ public protocol FileSysteming {
     /// - Parameter path: The path to the file or directory to remove.
     func remove(_ path: AbsolutePath) async throws
 
-    /// It removes the file or directory at the given path.
-    /// - Parameters:
-    ///   - path: The path to the file or directory to remove.
-    ///   - recursively: When removing a directory, it removes the sub-directories recursively.
-    func remove(_ path: AbsolutePath, recursively: Bool) async throws
```

Comment:
`FileManager` doesn't have a distinction for a recursive removal – and it's not something we're using in `tuist/tuist`